src/features/my-deposits/DepositsTable.tsx

```diff
@@ -27,13 +32,32 @@ export interface DepositsTableProps {
 }
 
 export const DepositsTable: FC<DepositsTableProps> = ({ account }) => {
```

Comment:
Deposits table and Pool table are v.similar. I think a Table component could be added at a later date to reduce the repetition. Same goes for the use of resize observer in each of the table files.

Reply:
Yeah 100% - need to make it more generic